### PR TITLE
layers: Fix checking enabled multiview feature

### LINKED ISSUE #4061: Multiview Validation not checking Vulkan11Features
**Describe the Issue**
Commit ef965275 added a new check to verify that the `multiview` feature is enabled when using `viewMask`.

However, Vulkan 1.1 made the Multiview feature core and rolled it into `VkPhysicalDeviceVulkan11Features`. The validation check here is only checking the older `VkPhysicalDeviceMultiviewFeatures` struct for the presence of the feature, which is causing a false positive if the user enables the feature using `VkPhysicalDeviceVulkan11Features`.

`layers/state_tracker.cpp` already has logic for copying multiview features into the Vulkan 1.1 features struct, so I believe all validation checks for multiview should be done against the `core11` features.

```diff
// layers/core_validation.cpp : 12974
- if (!enabled_features.multiview_features.multiview) {
+ if (!enabled_features.core11.multiview) {
```

**Valid Usage ID**
`VUID-VkSubpassDescription2-multiview-06558`

**Environment:**
 - OS: Windows
 - GPU: 
 - SDK or header version if building from repo: 1.3.212
 - Options enabled (synchronization, best practices, etc.):

**Additional context**

### PR DESCRIPTION
**Pull request by ziga-lunarg**
Closes #4061

### CODE REVIEW

File: tests/positive/render_pass.cpp [hunk 1]
```diff
@@ -995,4 +995,57 @@ TEST_F(VkPositiveLayerTest, RenderPassSingleMipTransition) {
...
+
+    vk::GetPhysicalDeviceFeatures2(gpu(), &features2);
+    if (vulkan_11_features.multiview == VK_FALSE) {
+        printf("%s multiview feature not supported, skipping test.\n", kSkipPrefix);
+        return;
+    }
+
+    ASSERT_NO_FATAL_FAILURE(InitState(nullptr, &features2));
+
+    m_errorMonitor->ExpectSuccess();
```
Comment (ncesario-lunarg) [inline tests/positive/render_pass.cpp:1028]: Might as well move this to the top of the test.

**Review by ncesario-lunarg — APPROVED** [review]
Minor nit, but LGTM!

File: tests/positive/render_pass.cpp [hunk 2]
```diff
@@ -995,4 +995,57 @@ TEST_F(VkPositiveLayerTest, RenderPassSingleMipTransition) {
     vk::DestroyFramebuffer(m_device->device(), fb, nullptr);
     vk::DestroyRenderPass(m_device->device(), rp, nullptr);
     vk::DestroySampler(m_device->device(), sampler, nullptr);
-}
\ No newline at end of file
+}
+
+TEST_F(VkPositiveLayerTest, CreateRenderPassWithViewMask) {
+    TEST_DESCRIPTION("Create render pass with view mask, but multiview feature disabled.");
```
Comment (Eearslya) [inline tests/positive/render_pass.cpp:1001]: Looks like a possible copy/paste error with this description?

Comment (ncesario-lunarg) [reply tests/positive/render_pass.cpp:1001]: ^ yep, nice catch!

**Review by ncesario-lunarg — CHANGES_REQUESTED** [review]

**Review by ncesario-lunarg — APPROVED** [review]

**Review by sjfricke — APPROVED** [review]
This fixes the original issue, but there are more `enabled_features.multiview_features` in the code that I think are also meant to be `core11`

**Pull request merged**